### ISSUE TITLE
Questions about module board positioning

### ISSUE THREAD

**Issue by murarduino** [solderparty/flexypin#4]
According to the recommendations in Readme. The edge of the module board should coincide with the center of the elliptical through hole.
To verify, I downloaded flexypin_adapters_hw and opened the PCB of ESP32-WROOM_flexypin for measurement. It was found that the center spacing of elliptical vias in the same row is 18.9726 mm, 
![image](https://github.com/solderparty/flexypin/assets/92354260/dfd8d6d5-1910-442f-9d08-9c46ad869b65)

while the data sheet of ESP32-WROOM module indicates that the width is 18+-0.15. 
![image](https://github.com/solderparty/flexypin/assets/92354260/ddeee261-5e6f-4390-b969-b5187940a483)

That makes me wonder. If I define a 12mm wide dual half hole module, how do I set the flexypin pad spacing?

**Comment by arturo182** [comment]
Sorry about the confusion!

The ESP32-WROOM Adapter was one of the first adapters designed, it was done before the FlexyPin spacing recommendation was established. Because of that, it doesn't quite follow the recommended alignment. 

However, FlexyPins are really springy and their performance isn't really affected much by having a bit more extra space in there.

If you're designing with FlexyPins currently, I'd recommend you follow the spacing recommendation :)

**Comment by lovefool** [comment]
I have a similar question about positioning. What do you mean "Module board edge"? 
Does it mean the edge of the board or the bottom of Castellated Hole? The Flexypin is springy, so maybe both works ok.

![Flexypin position 1](https://github.com/solderparty/flexypin/assets/3460081/e9834363-d30c-4b6d-b771-adba6d9ba631)

**Comment by arturo182** [comment]
Yes, I do mean the edge of the PCB, unless the Castellated Hole is really deep for some reason, then adjustments might have to be made.

**Comment by murarduino** [comment]
In fact, the core of the problem is worried about the elasticity of copper wire pins and the tightness of PCB edge size. The general pin cross-section diameter is 0.6mm, because the edge of the via (internal test) is indented to the PCB edge 0.3mm is no problem.
I use PCB test pins as copper pins

![71vHY+r9jVL _AC_SX679_](https://github.com/solderparty/flexypin/assets/92354260/8bb51b7c-b171-4fab-918d-4e8ed23fcf01)

[https://www.amazon.com/Black-Ceramic-Bead-Ring-Copper/dp/B00S4YKC8E](url)

Consider the elastic contraction of the pins (for this reason I also drew a 3D test pin with Fusion360 for simulation and analysis). To make sure I also consider setting the PCB (module) edge back another 0.3mm.

![image](https://github.com/solderparty/flexypin/assets/92354260/222957c2-0a14-4313-84aa-fcdd4e194e85)
